### ISSUE TITLE
Inference with point prompt

### ISSUE THREAD

**Issue by tinalmas** [med-air/3DSAM-adapter#23]
Dear Author, thanks for the nice work! My question is about the inference with point prompt. From the paper, ' For inference with point prompt, we calculate the center of the prompts and then crop a small patch surrounding this center, which has the same size as that for training, and do inference on the particular patch.' Do you mean that you only need to predict the segmentation result on one small patch where the point is in the center? What if the tumor is larger than the small patch?In this way, will the method be applicable to organ segmentation, since the organ could be much larger? Thanks!

**Comment by peterant330** [comment]
> Dear Author, thanks for the nice work! My question is about the inference with point prompt. From the paper, ' For inference with point prompt, we calculate the center of the prompts and then crop a small patch surrounding this center, which has the same size as that for training, and do inference on the particular patch.' Do you mean that you only need to predict the segmentation result on one small patch where the point is in the center? What if the tumor is larger than the small patch?In this way, will the method be applicable to organ segmentation, since the organ could be much larger? Thanks!

Yes, you are right. Actually, the size of the patch is a very important hyper-parameter. We select the patch size according to the size of tumors in the training set so that it can cover most of the cases. For organ segmentation, one way is to increase the patch size during the training, or you can use multiple prompts and conduct inference for each prompt (maybe using a sliding window inference method) and finally integrate all the predictions.
